### PR TITLE
Move 101 and 426 to Messaging

### PR DESCRIPTION
**Pull request by mnot**
for #335. Splitting this out for separate review.

### CODE REVIEW

File: draft-ietf-httpbis-messaging-latest.xml [hunk 1]
```diff
@@ -2024,7 +2024,7 @@ https://www.example.org
 </t>
 </section>
 
-<section title="Upgrade" anchor="header.upgrade">
+<section title="Upgrading Connections" anchor="header.upgrade">
```
Comment (reschke) [inline draft-ietf-httpbis-messaging-latest.xml:2027]: I would either leave the title as is, or insert a subsection that defines the field itself.

**Comment by mnot** [comment]
I suspect that we should be consistent regarding how we treat Upgrade and CONNECT, because both are extensions that use generic extensibility points, but have version-specific implementation, because they are peering into the actual connection management, not only expressing application semantics.

So maybe that means we define their general meaning in semantics, but their actual implementation in messaging (with the implication that h2 and h3 can do so if they choose to, but absent that, they can't be used there).

Does that make sense?

**Comment by reschke** [comment]
Sounds right to me.

**Comment by MikeBishop** [comment]
That seems reasonable, particularly as HTTP/2 and HTTP/3 already define how CONNECT is used.

**Comment by reschke** [comment]
I rebased and fixed the build. This is the first case where status code definitions would live outside the Semantics document (and that impacts IANA registration tables etc). My conclusion is that the status codes should stay in Semantics.

**Pull request closed without merging**